### ISSUE TITLE
Parameter to set minimum footlilne height

### ISSUE THREAD

**Issue by ghlecl** [jolars/moloch#22]
Hi.

First, thanks a lot for the work.

I am trying to switch over to Moloch from Metropolis.  I have been having issues with spacing: I prefer less dense slides.  Reading up on the blog post explaining the choices made in Moloch, watching the video about the theme at TUG 2024, and reading quite a few Stack Overflow questions, I have been able to "solve" (it is a matter of taste, so I feel "solve" is a bit strong, but ¯\\_(ツ)_/¯ ) most of my problems and get back to mostly the same look as Metropolis, but now in a more up-to-date theme.  I understand the choices and think they are the right ones: it was easier for me to make things less dense by adding `parskip` and `linespread`.

There is one thing though that has been really hard to solve.  The footline is not really slim.  When using this in with frame footnote, it makes the footnote appear really low on the slide, which in some rooms is problematic.  Some people will not see the bottom of the screen properly from the back.  I often sit in the back of rooms and I have this problem more often than people realize: writing at the bottom of slides is not always a good idea, so I usually like not to put too much things low on the slide.

This has been harder to solve than I thought.  The only way I found was to use this code in my preamble:
```latex
\setbeamertemplate{footline}{%
  \begin{beamercolorbox}[
      leftskip=4pt,%
      rightskip=5pt,%
      wd=\textwidth,%
      ht=0.55cm%      <========  THIS IS THE LINE I ADDED
    ]{footline}%
    \usebeamercolor[fg]{page number in head/foot}%
    \usebeamerfont{page number in head/foot}%
    \usebeamertemplate*{frame footer}%
    \hfill%
    \usebeamertemplate*{page number in head/foot}\vskip4pt%
  \end{beamercolorbox}%
}
```
which basically sets a minimum height to the footline.  The page number then remains low, but the foot note is slightly higher, which is what I want.  I usually prefer not to do this unless absolutely necessary, because then, on every update of the theme, I have to look if the definition has changed and rewrite this patch to keep up-to-date.

Here are my questions/suggestions:

1) Is there another way that I missed of achieving this, in which case, sorry for missing it.  I have searched for a few hours last night to no avail, but I never really tinkered much with themes, so I might have missed something obvious.

2) If there is no other way, again admitting I know little about themes and I might be asking for months of work which I do not want to impose on you, would it be hard for you to add a `\molochminfootheight` length to the theme that controls this minimum height, uses it in the footline definition and defaults to zero (or 1ex which I believe yields the current behavior)?

Thank you.

**Comment by samcarter** [comment]
As an easy workaround, you could add some space either before or after the footline template:

```
\documentclass{beamer}

\usetheme{moloch}
\addtobeamertemplate{footline}{\vskip2cm}{}

\begin{document}
\begin{frame}
test\footnote{text}
\end{frame}

\end{document}
```

**Comment by ghlecl** [comment]
Weird.  When I tried that your suggestion last night:
```latex
\addtobeamertemplate{footline}{\vskip0.25cm}{}
```
the page number kept moving up with the footnote, but after reading your comment, I tried again and it does not move.  I probably made a mistake in all my testing last night and did not realize that solution worked.  So sorry for wasting your time.

Thanks for that.  This is almost just as easy as having a dedicated length for it, so I will close this Github issue.  Sorry again for the confusion.
